### PR TITLE
Update release docs for 1.2.0

### PR DESCRIPTION
**Pull request by ts-nitish-a-jain**
# Description
Update release docs for 1.2.0. This is pending the management and update for testing and location feature.

## Links
[MINI-1253](https://jira.rakuten-it.com/jira/browse/MINI-1253?src=confmacro)

# Checklist
- [ ] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [ ] I removed all sensitive data before every commit, including API endpoints and keys
- [ ] I ran `fastlane ci` without errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+**Sample App**
+- *Feature:* Implementation of the `testing` Mini Apps SDK feature
+
+### 1.2.0 (2020-07-21)
+
+**SDK**
+- *Feature:* Possibility to use back and forward navigation inside MiniApp with SDK default UI or custom client provided UI
+- *Feature:* Ability to add a host app information string (RMAHostAppUserAgentInfo) that will get appended in the User agent.
+- *Feature:* Added support for `window.alert` JS dialogs in the mini app display
```
Comment (ts-leojoseph-r) [inline CHANGELOG.md:16]: Not just alert. Confirm and Prompt as well.

File: README.md [hunk 2]
```diff
@@ -220,36 +223,4 @@ See the *LICENSE* file for more info.
...
-### 1.1.0 (2020-06-01)
-
-- Added JavaScript bridge for passing data between Mini App and Host App. Your App now must implement `MiniAppMessageProtocol` in your view controller and provide the implementation when calling `MiniApp#create`.
-- Deprecated `MiniApp#create(appInfo:completionHandler:)`. Your App should instead use `MiniApp#create(appInfo:completionHandler:messageInterface)`.
-- Added `getUniqueId` function to `MiniAppMessageProtocol`. This function should provide a unique identifier (unique to the user and device) to Mini Apps.
-
-### 1.0.0 (2020-04-27)
-
-- Initial release
+See the full [CHANGELOG](CHANGELOG.md).
```
Comment (corycaywood) [inline README.md:226]: See my comment about the changelog on the JS SDK repo: https://github.com/rakutentech/js-miniapp/pull/25/files#r457785133

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+## CHANGELOG
+
+### X.Y.Z (YYYY-mm-dd)
```
Comment (corycaywood) [inline CHANGELOG.md:3]: This should be removed until the 1.2.0 release is complete.

File: README.md (hunk 2 above)
Comment (ts-nitish-a-jain) [reply README.md:226]: Let's converge it to that thread.

File: CHANGELOG.md (hunk 1 above)
Comment (ts-nitish-a-jain) [reply CHANGELOG.md:16]: Okay. Updated. Thanks.

File: CHANGELOG.md (hunk 3 above)
Comment (ts-nitish-a-jain) [reply CHANGELOG.md:3]: This has been removed.

File: README.md [hunk 4]
```diff
@@ -6,15 +6,17 @@
...
 
-This open-source library allows you to integrate a Rakuten MiniApp into your iOS applications. It also provides more features to interact with the native application via bridge.
+This open-source library allows you to integrate Mini App ecosystem into your iOS applications. 
+Mini App SDK also facilitates communication between a mini app and the host app via a message bridge.
 
 ## Features
 
 - Load MiniApp list
-- Load MiniApp meta-informations
+- Load MiniApp metadata
```
Comment (khairul-alam-licon) [inline README.md:15]: Good choice of the keyword.

**Pull request merged**